README.md

```diff
@@ -81,6 +84,8 @@ One great use case for this is where you have an API that you want to be JSON co
 For example:
 
 ```js
+import superjson from 'superjson';
```

Comment:
I don't think the superjson imports add to understanding. Could you remove them?